MvvmCross-Forms/MvvmCross.Forms.Droid/Views/MvxFormsAppCompatActivity.cs

```diff
@@ -14,19 +14,19 @@
 using MvvmCross.Forms.Platform;
 using MvvmCross.Forms.Views;
 using MvvmCross.Platform;
-using MvvmCross.Platform.Droid.Platform;
-using Xamarin.Forms.Platform.Android;
 
 namespace MvvmCross.Forms.Droid.Views
 {
     public class MvxFormsAppCompatActivity : MvxEventSourceFormsAppCompatActivity, IMvxAndroidView
     {
         private View _view;
+        private Assembly _resourceAssembly;
 
         protected MvxFormsAppCompatActivity()
         {
             BindingContext = new MvxAndroidBindingContext(this, this);
             this.AddEventListeners();
+            _resourceAssembly = Assembly.GetCallingAssembly();
```

Comment:
Obtain the calling assembly here so you don't need to override OnCreate in the derived activity class.